lib/analog_clock.dart

```diff
@@ -51,6 +53,7 @@ class AnalogClock extends StatefulWidget {
       showNumbers = true,
       showAllNumbers = false,
       showSecondHand = true,
+      useMilitaryTime = false,
```

Comment:
Also I think this should be `true` by default.

Reply:
Done!